openapi/data_repository_service.swagger.yaml

```diff
@@ -194,57 +181,85 @@ definitions:
           sha512
   Bundle:
     type: object
-    required: ['id', 'object_ids', 'created', 'updated', 'version', 'checksums']
     properties:
       id:
         type: string
-        description: |-
+        description: >-
           An identifier, unique to this Data Bundle
-      object_ids:
-        type: array
-        items:
-          type: string
+      name:
+        type: string
         description: |-
-          The list of Data Objects that this Data Bundle contains.
+          A string that can be optionally used to name a Data Bundle.
+      size:
+        type: string
+        format: int64
+        description: |-
+          The computed cumulated size of all Data Objects in bytes.
```

Comment:
Is this recursive? In other words, if the Bundle contains Bundles that contain Objects, is `size` expected to be based on walking the whole tree? Either way, let's clarify the description. (And nit -- I think "computed cumulated" should be "cumulative" or "total".)

Reply:
@dglazer Recursive

@ddietterich Providing this would give the user a handle to the total size of the Bundle they will be downloading. If having `access_methods` are included in the Bundle specification then having `size` is a must.

Reply:
Clarified Bundle > size in description. size represents only the cumulative size of all objects within the Bundle's contents field (so depth 1)